app/forms/onboarding/request_form.rb

```diff
@@ -39,7 +39,9 @@ def proxy_base_url
 
   def uri
     return unless model.api_backend
-    URI.join(api_base_url, path || SLASH).to_s
+    base_url_path = api_base_url.respond_to?(:path) ? api_base_url.path : nil
```

Comment:
Isn't it exactly the same as this? 👇 
```suggestion
    base_url_path = api_base_url.try(:path)
```